vignettes/saving_serializing.Rmd

```diff
@@ -297,7 +317,7 @@ associated with the model: - The layers' weights - The optimizer's state - Any v
 associated with stateful model metrics (if any).
 
 ```{r}
-save_model_weights_tf(model, "my_weights")
+save_model_weights_hdf5(model, "my_weights.h5")
```

Comment:
Now I'm confused... didn't we say

> The recommended way to save a subclassed model is to use `save_model_weights_tf` to 
create a TensorFlow SavedModel checkpoint, which will contain the value of all variables 
associated with the model:

?

Reply:
Oh sorry, I change it to test if it worked and forgot to change back :)

Reply:
ah okay, I'm relieved it wasn't some fundamental misunderstanding from my side ;-))
... and does it work (I guess not)?